_posts/2020-12-24-newpipe-0.20.7.md

```diff
@@ -0,0 +1,61 @@
+
+In another attempt to improve the ReCaptcha experience, [@Stypox](https://github.com/Stypox) fixed a situation where only a blank page was shown. They also prevented device information from being leaked inside the user agent used for the ReCaptcha service, and are likely very exhausted with that topic at this point. We thank them massively for taking up this issue time and time again.
+
+Talking about time - when sharing video URLs from services other than YouTube, the app doesn't try to append a timestamp to the URL anymore, which used to make them invalid. In the future, we hope to support the timestamp formats of every platform, but for now, you get non-timestamped URLs that work. This was provided by [@B0pol](https://github.com/B0pol). Thank you!
+
+The last big fix is brought to you by [@TobiGr](https://github.com/TobiGr), who got YouTube Music's search to work once again. Search all the music that you want now (with the filters in the 3-dot menu next to the search bar) and enjoy listing to Christmas songs that you cannot get out of your head.
+
+## Nerd talk
+
+There were quite a few very technical changes under the hood in this release. We decided to provide you this nice list with changes not affecting you at all, because we still want to publicly thank their creators:
+
+- The Invidious (an alternative YouTube frontend) URL list in the manifest was updated by [@opusforlife2](https://github.com/opusforlife2)
+- Our CI tests were switched from Travis to GitHub by [@FireMasterK](https://github.com/FireMasterK)
+- Some internal variables and comments had typos, and [@mhmdanas](https://github.com/mhmdanas) fixed them
+- We use `ContextCompat.getDataDir()`, `ServiceCompat.stopForeground()` and `ActivityCompat.recreate()` now, because [@Isira-Seneviratne](https://github.com/Isira-Seneviratne) decided that is good and everyone agreed, and also because older Android versions can now use the existing code without needing special caveats and corner case handling
+- [@Isira-Seneviratne](https://github.com/Isira-Seneviratne) changed how the file hash calculation in the Downloads menu is displayed - it does not use ProgressDialog anymore because that is deprecated
+- The database export logic got its own class so that we can create unit tests for it, thanks to [@XiangRongLin](https://github.com/XiangRongLin)
+- [@Isira-Seneviratne](https://github.com/Isira-Seneviratne) is on fire this release; they also changed Kotlin synthetics to view bindings because of deprecation of the former
+- The NewPipe extractor supports stream segments for YouTube now, which has been included in this release, because of [@vkay94](https://github.com/vkay94)
```

Comment:
```suggestion
- The NewPipe extractor supports stream segments for YouTube now, which has been included in this release, because of [@vkay94](https://github.com/vkay94). [@vkay94](https://github.com/vkay94) is currently working on integrating the changes into the app.
```

Reply:
Oh. If it's not there in the front-end, we should just remove it altogether. When the UI PR is merged, this can be added in one go.

Reply:
I agree